modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
mmm... so I guess you develop with the static html that mocks Liferay Utils. Should this be uploaded? I'm a bit worried about this way of developing because it can mask a lot of bugs/things that happens only in the portal because of mocking everything, adding atlas directly... 

What did you find lacking of the "normal" way?

Reply:
I see... I read the comment in https://github.com/brianchandotcom/liferay-portal/pull/102952.

> We are making this to cut all the functionalities in different and more reviewable prs. We need icon file, html file and scss file to keep developing all the frontend part before we wrap it all in a jsp component and at the same time have can test all the real apis endpoint. We should remove all the static env files before step 4 of 5, when the module should be ready and startable. We have been addressing this issue lately trying to find a common solution to be able to easily develop frontend components following the liferay patterns. For this reason at this time this module it’s hidden from liferay startable modules.

But I don't understand why you can't create a react-template module that just returns a react app and work from there?

Reply:
Thank you @nhpatt. Do we already have some example within portal where we use a react-module-template? We just need tests and static env support

Reply:
The react template in blade (https://help.liferay.com/hc/es/articles/360018164591-Blade-CLI) will create a portlet that returns a react application. But it's not a static environment, has to be deployed in a liferay server. But it can be a fast deployment by using -a flag or even deployFast.

I've worked with a static environment in liferay before... but I haven't uploaded it. Do you need to share it with other team members?